CMakeLists.txt

```
@@ -244,8 +244,8 @@ if(WIN32)
 endif(WIN32)
 
 if(APPLE)
-    set(MACDEPLOYQT macdeployqt)
-    set(MACDEPLOYQT_ARGS ${CMAKE_CURRENT_BINARY_DIR}/${APP_NAME}.app -dmg)
+    set(MACDEPLOYQT ${Qt5Widgets_DIR}/../../../bin/macdeployqt)
+    set(MACDEPLOYQT_ARGS ${APP_NAME}.app -dmg -qmldir=../client/qml)
```

```suggestion
    set(MACDEPLOYQT_ARGS ${APP_NAME}.app -dmg -qmldir="${PROJECT_SOURCE_DIR}/client/qml")
```
because you cannot rely on the build directory always being one under the sources directory.